src/lib/seam/components/DeviceDetails/DeviceDetails.tsx

```diff
@@ -36,12 +37,13 @@ export function DeviceDetails({
     return null
   }
 
-  const props = {
+  const props: Omit<DeviceDetailsProps, 'deviceId'> = {
```

Comment:
What is this needed now?

Reply:
It took me a minute to figure out why the prop wasn't being passed through correctly—adding strict typing here will save others this unnecessary delay/investigation. Not required, but not harmful, I think,

Reply:
Ah ok, I expanded the diff context and I see what's up 👍🏻